src/Components/Actions/ActionsTabItem.js

```diff
@@ -14,19 +17,43 @@ export default function ActionsTabItem({ actionItem, index }) {
     />
   );
 
-  const actionEndingTime = Math.floor(actionItem.endingTime / 100000);
-  //actionItem.endingTime is a unix timestamp regarding it's end,
-  //the division and floor are just for the large int to be small enough to fit in the screen
-  //Need to create dynamic counter for this and replace it.
+
+  // * When ending time is updated, set the format in seconds.
+  useEffect(() => {
+    if(typeof actionItem.endingTime === 'number' && typeof actionItem.startingTime === 'number')
```

Comment:
Is this check needed?
I truly wonder how we should approach this kind of things, so that we'll have a norm.

Reply:
Yes. This one is needed. at first, before getting the information, the variable may contain undefined value.
I can make a general function that check if a variable is a number or not.

Reply:
this section has been removed.